pkg/ui/workspaces/db-console/src/redux/alerts.ts

```diff
@@ -707,6 +695,105 @@ export const licenseTypeSelector = createSelector(
   data => licenseTypeNames.get(data.LicenseType) || "None",
 );
 
+export const licenseUpdateDismissedLocalSetting = new LocalSetting(
+  "license_update_dismissed",
+  localSettingsSelector,
+  moment(0),
+);
+
+const licenseUpdateDismissedPersistentLoadedSelector = createSelector(
+  (state: AdminUIState) => state.uiData,
+  uiData => uiData && has(uiData, LICENSE_UPDATE_DISMISSED_KEY),
+);
+
+const licenseUpdateDismissedPersistentSelector = createSelector(
+  (state: AdminUIState) => state.uiData,
+  uiData => {
+    return (
+      (uiData &&
+        uiData[LICENSE_UPDATE_DISMISSED_KEY] &&
+        uiData[LICENSE_UPDATE_DISMISSED_KEY].data &&
```

Comment:
nit: i think this can be collapsed a little:
```
    return moment(uiData?.[LICENSE_UPDATE_DISMISSED_KEY]?.data ?? 0);
```

Reply:
ooh nice. thanks!